### PR TITLE
staketia migration - disable staketia LS

### PR DESCRIPTION
**Pull request by sampocs**
During the stTIA migration, liquid stakes are disabled from staketia and will be handled in stakeibc. This PR removes the `LiquidStake` functionality from staketia

### CODE REVIEW

File: x/staketia/keeper/msg_server.go [hunk 1]
```diff
@@ -25,12 +26,7 @@ var _ types.MsgServer = msgServer{}
 
 // User transaction to liquid stake native tokens into stTokens
 func (k msgServer) LiquidStake(goCtx context.Context, msg *types.MsgLiquidStake) (*types.MsgLiquidStakeResponse, error) {
-	ctx := sdk.UnwrapSDKContext(goCtx)
-	stToken, err := k.Keeper.LiquidStake(ctx, msg.Staker, msg.NativeAmount)
-	if err != nil {
-		return nil, err
-	}
-	return &types.MsgLiquidStakeResponse{StToken: stToken}, nil
+	return nil, errors.New("Liquid staking is no longer enabled in staketia")
```
Comment (shellvish) [inline x/staketia/keeper/msg_server.go:29]: _very_ nit, should this message say "use stakeibc instead"?

**Review by shellvish — COMMENTED** [review]
This looks great, thank you for doing this!

**Comment by riley-stride** [comment]
Should we also pull LiquidStake out of:
- msgs.go
- msgs_test.go
A few other files, I see some mentions we could prob clean up when searching "LiquidStake" in x/staketia

**Comment by riley-stride** [comment]
Otherwise this lgtm!

File: x/staketia/keeper/msg_server.go (hunk 1 above)
Comment (sampocs) [reply x/staketia/keeper/msg_server.go:29]: done!

**Comment by sampocs** [comment]
@riley-stride we definitely could remove them from msgs but I was trying to keep the changes to a minimum and I don't think there's any harm in keeping them there since we'll eventually deprecate the whole module!

**Comment by riley-stride** [comment]
> @riley-stride we definitely could remove them from msgs but I was trying to keep the changes to a minimum and I don't think there's any harm in keeping them there since we'll eventually deprecate the whole module!

Makes sense. Good point about deprecating the module. I'm signed off here.

**Pull request merged**